src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs

```diff
@@ -75,7 +75,7 @@ public JaegerExporterOptions()
         public int? MaxPayloadSizeInBytes { get; set; } = DefaultMaxPayloadSizeInBytes;
 
         /// <summary>
-        /// Gets or sets the export processor type to be used with Jaeger Exporter.
+        /// Gets or sets the export processor type to be used with Jaeger Exporter. The default value is Batch.
```

Comment:
Is there a way to make it a [cref](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/cref-attribute)?

Reply:
Should be.. @benbhall Could you do that change?

Reply:
Yep, on it.